scalafix-tests/output/src/main/scala/test/AnchorPatches.scala

```diff
@@ -9,9 +9,8 @@ import scala.collection.mutable.{ // scalafix:ok RemoveUnusedImports
 
 
 import scala.concurrent.duration.Duration
-import scala.concurrent.duration.FiniteDuration
 
 object AnchorPatches {
-  def d1s: FiniteDuration = Duration(1, "s")
-  def d2s = Duration(2, "s") // scalafix:ok ExplicitResultTypes
+  
+  private def d2s = Duration(2, "s") // scalafix:ok RemoveUnused
```

Comment:
rewrote the test to leverage RemoveUnused only, as ExplicitResultTypes is not available on Scala 3